.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2
 jobs:
   build:
     docker:
-    - image: circleci/golang:1.14.10
+    - image: cimg/go:1.19.4
```

Comment:
why not using circleci/golang?

Reply:
circleci/golang is no longer updated.
cimg/go is designed to supercede the legacy CircleCI Go image, circleci/golang.
see:  https://github.com/CircleCI-Public/cimg-go

Reply:
circleci working_directory has changed from /go to /home/circleci/go.